src/System.Collections.Specialized/src/System/Collections/Specialized/NameValueCollection.cs

```diff
@@ -17,11 +17,10 @@ namespace System.Collections.Specialized
     /// <para>Represents a sorted collection of associated <see cref='System.String' qualify='true'/> keys and <see cref='System.String' qualify='true'/> values that
     ///    can be accessed either with the hash code of the key or with the index.</para>
     /// </devdoc>
-    [Serializable]
     public class NameValueCollection : NameObjectCollectionBase
     {
-        private String[] _all;
-        private String[] _allKeys;
+        private String[] _all; // Do not rename (binary serialization)
```

Comment:
Nit: not worth changing just for this. These comments are no longer needed: the whole file can roll back.